nkms/blockchain/eth/sol/compile.py

```diff
@@ -1,33 +1,32 @@
-import glob
+import itertools
 import os
 
-from os.path import join, dirname, abspath
-from solc import compile_files
-from solc import install_solc
-
-from nkms.blockchain import eth
-from nkms.config.configs import _DEFAULT_CONFIGURATION_DIR
-from tests.blockchain.eth import contracts
+from os.path import abspath, dirname
+from solc import install_solc, compile_files
 
 
 class SolidityCompiler:
-    __default_version = 'v0.4.21'
-    __default_configuration_path = os.path.join(_DEFAULT_CONFIGURATION_DIR, 'compiler.json')
-    __default_sol_binary_path = os.path.join(os.environ['VIRTUAL_ENV'], 'bin', 'solc')   # TODO: Does not work with pytest w/o intervention
+    __default_version = 'v0.4.21'  # TODO: Update to 4.22
+    __default_configuration_path = os.path.join(dirname(abspath(__file__)), './compiler.json')
 
-    def __init__(self, solc_binary_path=None, configuration_path=None):
-        solc_binary_path = solc_binary_path if solc_binary_path is not None else self.__default_sol_binary_path
-        configuration_path = configuration_path if configuration_path is not None else self.__default_configuration_path
+    __default_sol_binary_path = os.path.join(os.environ['VIRTUAL_ENV'], 'bin', 'solc')  # TODO: Does not work with pytest w/o intervention
```

Comment:
Maybe this?
```
__bin_path = os.path.dirname(distutils.spawn.find_executable('python'))
__default_sol_binary_path = os.path.join(__bin_path, 'solc')
```

Reply:
`__bin_path` is `/your/virtual/environment/bin`

Reply:
Neat. :heavy_check_mark: